src/PixiParser.Benchmarks/Program.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using BenchmarkDotNet.Running;
 
-namespace PixiEditor.Parser.Benchmarks
+namespace PixiEditor.Parser.Benchmarks;
+
+internal class Program
 {
-    class Program
+    static void Main()
     {
-        static void Main()
-        {
-            BenchmarkRunner.Run<Benchmarks>();
-        }
+        BenchmarkRunner.Run<Benchmarks>();
     }
 }
```

Comment:
I think you could just make this a top-level statement program file

Reply:
Indeed! I totally forgot that 🤔